src/libdrakvuf/drakvuf.c

```diff
@@ -879,3 +879,49 @@ bool drakvuf_set_vcpu_gprs(drakvuf_t drakvuf, int vcpu, registers_t* regs)
 
     return xen_set_vcpu_ctx(drakvuf->xen, drakvuf->domID, vcpu, &ctx);
 }
+
+static bool is_valid_vcpu(drakvuf_t drakvuf, unsigned int vcpu)
+{
+    // VMs with more than 16 vCPUs are not supported for usage with IPT
+    // this limit is DRAKVUF specific
+    return vcpu < 16 && drakvuf->vcpus > vcpu;
```

Comment:
So I think we should declare `16` as a #define in `private.h` as it is now being used at more spots, say `#define MAX_DRAKVUF_VCPU 16`. That way we can choose to increase it any time fairly easily in the future. It is a random number at the moment, there is nothing technically limiting us to 16, it's just I think a high enough number that will be unlikely to get exceeded for any reasonable usecase.

Reply:
ok

Reply:
done